spec/features/account_page_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe "Account page" do	
+	before do
+		# Sign in
+		User.create!(username: '-', email: 'user@example.com', password: 'password')
+		visit '/sign_in'
+		fill_in 'Email', with: 'user@example.com'
+		fill_in 'Password', with: 'password'
+		click_button 'Sign in'
+
+		# Go to account page
+		visit '/account'
+	end
+
+	it "allows user to edit his profile" do
```

Comment:
You can also add some checks to see if the data was saved correctly. For example, create a user with a certain name, email, and `digest_subscriber = true`, then after clicking save, reload the user record and check that the values are correct.